signadot/operator/README.md

```diff
@@ -132,3 +134,13 @@ style resources and are not needed in an installation which uses the new
 | `tunnel.auditor.inboundRulesLuaScript`   | All inbound traffic (from cluster to workstation) will pass thru this script (if defined) in the Envoy auditor, check [HTTP Lua filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/lua_filter#stream-handle-api) documentation for details  | `""`    |
 | `tunnel.auditor.outboundRulesLuaScript`  | All outbound traffic (from workstation to cluster) will pass thru this script (if defined) in the Envoy auditor, check [HTTP Lua filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/lua_filter#stream-handle-api) documentation for details | `""`    |
 |                                          |                                                                                                                                                                                                                                                                              |         |
+### Istio Parameters
+
+When Istio is enabled (`istio.enabled: true`), the Signadot Operator manipulates Istio VirtualServices by applying new HTTPRoutes where appropriate to direct traffic to sandboxed workloads. You can configure the operator to add labels and annotations to these objects when they are in use by the operator.  Note that these labels and annotations are only added if not present when the object comes into use. This can be useful for temporarily disabling CI sync, amongst other possibilities.
+
+
+| Name                          | Description                                        | Default |
+| ----------------------------- | -------------------------------------------------- | ------- |
+| `istio.enabled`               | Enable Istio integration                           | `false` |
+| `istio.additionalAnnotations` | Annotations to add to istio objects if not present | `{}`    |
```

Comment:
I think we can be more specific that this is `VirtualService` objects that are being modified by the signadot controllers that will get these labels and annotations.

Reply:
Ah, I see that there's the explanation below in the helm values - we can copy the same here.

Reply:
updated to mention VirtualServices explicitly.